azimuth/utils/ml/postprocessing.py

```diff
@@ -18,6 +19,7 @@ def is_multiclass(self):
         return np.allclose(self.probs.sum(-1), 1.0)
 
     def __getitem__(self, item: int) -> "PostProcessingIO":
+        """Useful to get from a batch result to a single utterance result."""
```

Comment:
I think we can skip the "Useful to", and maybe invert the sentence? That's candy, though.
```suggestion
        """Get a single utterance result from a batch result."""
```